icechunk/src/repository.rs

```diff
@@ -796,6 +798,20 @@ impl Repository {
         Ok(session)
     }
 
+    pub async fn rewrite_manifests(
+        &self,
+        branch: &str,
+        message: &str,
+        properties: Option<SnapshotProperties>,
+    ) -> RepositoryResult<SnapshotId> {
+        let mut session = self.writable_session(branch).await?;
+
+        session
```

Comment:
Will this rewrite every array's manifest? would it be useful to let them select a list of arrays? What happens if there is nothing to do (say because the are no new chunks in the new splits) will a commit still happen?

Reply:
Taking the branch is quite strange. Maybe we should only have the session method? Actually, a writable session method?

Reply:
Interesting, we actually have an `ops` module for that, I guess you can put it in there. But bottom line, the important thing is where we add it in the python interface.